package.json

```diff
@@ -35,10 +35,10 @@
   },
   "peerDependencies": {
     "cldr-data": ">=25",
-    "globalize": ">=1.1.0-rc.5"
+    "globalize": "^1.1.0"
   },
   "devDependencies": {
-    "babel": "^5.6.14",
+    "babel": "^6.3.26",
```

Comment:
Thank you all for your comments. Since this isn't a blocker for this PR, I've moved this into #13